app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="container">
+  <h1 class="display-5">Public Recipes</h1>
+  <ul class="list-group">
+    <% @recipes.each do |recipe| %>
+      <li class="list-group-item d-flex justify-content-between">
+        <div class="col-4">
+          <%= link_to recipe.name, recipe_path(recipe.id), class: "text-decoration-none text-dark fw-bold me-3" %>
+          <p class="mb-0">By: <%= recipe.user.name %></p>
+        </div>
+        <div class="col-7">
+          <p class="mb-0"><%= recipe.description %></p>
+        </div>
```

Comment:
- [ ] You are displaying the recipe `description ` on the right side of the public recipes. However, you are suppose to display `Total food items: ` and `Total price:  `. Kindly update your design to reflect the actual design. You should also consider adding a margin and darker border around each individual public recipe card as shown in the original design :point_down:

### Your design 


![Screenshot from 2023-04-28 23-00-16](https://user-images.githubusercontent.com/65233860/235267678-dc4b6e9a-a52e-4471-877d-32fabae4b4d5.png)

### Actual design 

![Screenshot from 2023-04-28 22-45-54](https://user-images.githubusercontent.com/65233860/235267705-9dbf89a1-99b8-47a6-a921-82be981721bd.png)

Reply:
![13](https://user-images.githubusercontent.com/90111138/235280575-835f1aa7-9519-4040-85fd-93d4a804d413.JPG)